python/paddle/fluid/layers/control_flow.py

```diff
@@ -1912,44 +1912,28 @@ def __call__(self):
 
 class DynamicRNN(object):
     """
-    The dynamic RNN can process a batch of sequence data. The length of each
-    sample sequence can be different. This API automatically process them in
-    batch.
+    **Note: the input of this class should be LoDTensor which holds the
+    information of variable-length sequence. If the input is Tensor,
+    please use StaticRNN (fluid.layers.** :ref:`api_fluid_layers_StaticRNN` **).**
```

Comment:
Now our python-level Tensor is also LoDTensor? I guess we can say "if the input is fixed-length Tensor, please use StaticRNN for better performance" ?

Reply:
Done.